core/web/resolver/query.go

```diff
@@ -221,18 +221,13 @@ func (r *Resolver) Features(ctx context.Context) (*FeaturesPayloadResolver, erro
 	return NewFeaturesPayloadResolver(r.App.GetConfig()), nil
 }
 
-// Node retrieves a node by ID
-func (r *Resolver) Node(ctx context.Context, args struct{ ID graphql.ID }) (*NodePayloadResolver, error) {
+// Node retrieves a node by ID (Name)
+func (r *Resolver) Node(ctx context.Context, args struct{ Name string }) (*NodePayloadResolver, error) {
```

Comment:
Same comment regarding possibly keeping `graphql.ID` as the type, for consistency.